[FEATURE] new fury distribution v1.23.1 (#15)
sighupio/fury-distribution: docs/releases/v1.23.1.md

# Kubernetes Fury Distribution Release v1.23.1

Welcome to the KFD release `v1.23.1`. This is a patch release
fixing bugs in all the core modules.

The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each
core module listed below along with the upgrade path of the distribution.

> If upgrading from `v1.23.0`, you **must** delete all the indicated objects (StatefulSet, Deployment, DaemonSet, etc)

Reviewer: ```suggestion

⚠️ If upgrading from `v1.23.0`, you **must** delete all the indicated objects (StatefulSet, Deployment, DaemonSet, etc)
```

- [monitoring](https://github.com/sighupio/fury-kubernetes-monitoring) 📦 core module: v1.14.0 -> [**v1.14.1**](https://github.com/sighupio/fury-kubernetes-monitoring/releases/tag/v1.14.1)
  - No updates on the components of the module
  - `commonLabels` bugfix
- [logging](https://github.com/sighupio/fury-kubernetes-logging) 📦 core module: v1.10.0 -> [**v1.10.2**](https://github.com/sighupio/fury-kubernetes-logging/releases/tag/v1.10.2)
  - No updates on the components of the module
  - `commonLabels` bugfix
- [ingress](https://github.com/sighupio/fury-kubernetes-ingress) 📦 core module: v1.12.0 -> [**v1.12.2**](https://github.com/sighupio/fury-kubernetes-ingress/releases/tag/v1.12.2)
  - Update [forecastle] from version `1.0.73` to `1.0.75`.
  - `commonLabels` bugfix
- [dr](https://github.com/sighupio/fury-kubernetes-dr) 📦 core module: v1.9.0 -> [**v1.9.2**](https://github.com/sighupio/fury-kubernetes-dr/releases/tag/v1.9.2)
  - No updates on the components of the module
  - `commonLabels` bugfix
- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: v1.6.0 -> [**v1.6.2**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.2)

>   - No updates on the components of the module

Reviewer: ```suggestion
  - Fixed an issue present only in `v1.6.0` with a missing volume mount that broke the audit process (policy enforcement was unaffected)
```

  - `commonLabels` bugfix

> Please refer the individual release notes for detailed information

## Upgrade path

### Katalog Procedure

To upgrade this distribution from `v1.7.x` to `v1.23.1`, you need to download this new version, vendor the dependencies,
finally applying the `kustomize` project.

```bash
furyctl vendor -H
kustomize build . | kubectl apply -f -
```

> **NOTE**: *The upgrade takes some minutes (depends on the cluster size), and you should expect some downtime during
the upgrade process.*

### Terraform Procedure

## Test it

If you want to test the distribution in a test environment, spin up a
[`kind`](https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.0) cluster, then deploy all rendered manifests.

```bash
$ kind version

> kind v0.11.0 go1.16.4 darwin/amd6

Reviewer: ```suggestion
kind v0.11.0 go1.16.4 darwin/amd64
```

> $ curl -Ls https://github.com/sighupio/fury-distribution/releases/download/v1.23.0/katalog/tests/config/kind-config | kind create cluster --image registry.sighup.io/fury/kindest/node:v1.23.0 --config -

Reviewer: ```suggestion
$ curl -Ls https://github.com/sighupio/fury-distribution/releases/download/v1.23.1/katalog/tests/config/kind-config | kind create cluster --image registry.sighup.io/fury/kindest/node:v1.23.1 --config -
```

# Kubernetes Fury Distribution Release v1.23.1

Welcome to the KFD release `v1.23.1`. This is a patch release
fixing bugs in all the core modules.

The team has been working to make the release upgrade as simple as possible, so read carefully the upgrade path of each
core module listed below along with the upgrade path of the distribution.

> ⚠️ If upgrading from `v1.23.0`, you **must** delete all the indicated objects (StatefulSet, Deployment, DaemonSet, etc)

Reviewer: ```suggestion
⚠️ If upgrading from `v1.23.0`, you **must** delete all the objects (StatefulSet, Deployment, DaemonSet, etc) as specified in the release notes of the modules
```

- [logging](https://github.com/sighupio/fury-kubernetes-logging) 📦 core module: v1.10.0 -> [**v1.10.2**](https://github.com/sighupio/fury-kubernetes-logging/releases/tag/v1.10.2)
  - No updates on the components of the module
  - `commonLabels` bugfix
- [ingress](https://github.com/sighupio/fury-kubernetes-ingress) 📦 core module: v1.12.0 -> [**v1.12.2**](https://github.com/sighupio/fury-kubernetes-ingress/releases/tag/v1.12.2)
  - Update [forecastle] from version `1.0.73` to `1.0.75`.
  - `commonLabels` bugfix
- [dr](https://github.com/sighupio/fury-kubernetes-dr) 📦 core module: v1.9.0 -> [**v1.9.2**](https://github.com/sighupio/fury-kubernetes-dr/releases/tag/v1.9.2)
  - No updates on the components of the module
  - `commonLabels` bugfix
- [OPA](https://github.com/sighupio/fury-kubernetes-opa) 📦 core module: v1.6.0 -> [**v1.6.2**](https://github.com/sighupio/fury-kubernetes-opa/releases/tag/v1.6.2)
  - Fixed an issue present only in `v1.6.0` with a missing volume mount that broke the audit process (policy enforcement was unaffected)
  - `commonLabels` bugfix

> Please refer the individual release notes for detailed information

## Upgrade path

### Katalog Procedure

> 

Reviewer: ```suggestion
To upgrade the distribution from `v1.23.0` to `v1.23.1` please follow the instructions written in the release notes of each core module.

```
